### PR TITLE
Update read me

### PR DESCRIPTION
**Pull request by jim-at-jibba**
Thanks again for your help. 

Here is the updated read me.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,31 @@
...
-
-1. Open src/standalone.ts and update your project directory paths
-2. Start react native metro bundler and make sure its running on 8081
-3. After running tsc on this directory once, run node src/standalone.js
-4. Now run the app in android emulator, it would wait on a white screen for debugger to attach, if its not a white empty screen then something didn't go well
-5. Now on neovim, setup your breakpoints and run dap continue with below config
+## Install
 
+```sh
+git clone git@github.com:jim-at-jibba/nvim-dap-reactnative.git
```
Comment (sultanahamer) [inline README.md]: Should I update the link here to this repo or let it be to your forked repo?

Comment (jim-at-jibba) [reply README.md]: Sorry that was a mistake. It should be yours. I will update now. Sorry

File: README.md [hunk 2]
```diff
@@ -31,10 +45,52 @@ M.config = {
...
 	},
 }
+
+-- OR
+-- directly - typescript react example
+
+dap.adapters.node2 = {
+	type = "executable",
+	command = "node",
+	args = { os.getenv("HOME") .. "/code/utils/vscode-node-debug2/out/src/nodeDebug.js" },
```
Comment (sultanahamer) [inline README.md]: Users might get confused with the path here. Let this be some path where we don't have any custom values

like may be `os.getenv("HOME") .. "/vscode-node-debug2/out/src/nodeDebug.js"`

Comment (jim-at-jibba) [reply README.md]: Yep that makes sense.

**Comment by jim-at-jibba** [comment]
I have pushed updates to the changes above. Thanks for looking so quickly

File: README.md [hunk 3]
```diff
@@ -1,17 +1,31 @@
-# nvim-dap-reactnative
-Standalone debug adapter protocol for react native. Hopefully won't have any specific editor/ide stuff
+<h1 align="center">Welcome to NVIM Dap React Native 👋</h1>
+<p>
+  <img alt="Version" src="https://img.shields.io/badge/version-1.9.1-blue.svg?cacheSeconds=2592000" />
+  <a href="https://github.com/microsoft/vscode-react-native#readme" target="_blank">
+    <img alt="Documentation" src="https://img.shields.io/badge/documentation-yes-brightgreen.svg" />
+  </a>
+  <a href="https://github.com/microsoft/vscode-react-native/graphs/commit-activity" target="_blank">
+    <img alt="Maintenance" src="https://img.shields.io/badge/Maintained%3F-yes-green.svg" />
```
Comment (sultanahamer) [inline README.md]: As we are not actively maintaining yet, I am not comfortable having this maintained as `Yes` as of now. If we keep using this everyday and keep fixing issues people face for like few months, we can call this maintained. Else we don't know at all if something is broken.

Comment (jim-at-jibba) [reply README.md]: I have removed the badges as they are not correct.

File: README.md [hunk 4]
```diff
@@ -1,17 +1,31 @@
...
+  </a>
+  <a href="https://github.com/microsoft/vscode-react-native/blob/master/LICENSE" target="_blank">
+    <img alt="License: %reactNative.license%" src="https://img.shields.io/github/license/sultanahamer/NVIM Dap React Native" />
```
Comment (sultanahamer) [inline README.md]: This link seems broken. Also this should be released in the same license terms as Microsoft's VSCode's original plugin I guess.

We can add this file in our codebase and it should work I guess

https://github.com/microsoft/vscode-react-native/blob/master/LICENSE.txt

Comment (jim-at-jibba) [reply README.md]: I have removed the Licence badge too. I am not sure about the licence. That is up to you

**Comment by sultanahamer** [comment]
Thanks for the help on the PR. It looks real good now.

**Pull request merged**